homeassistant/components/opensky/sensor.py

```diff
@@ -117,14 +117,20 @@ def _handle_boundary(self, flights, event, metadata):
         for flight in flights:
             if flight in metadata:
                 altitude = metadata[flight].get(ATTR_ALTITUDE)
+                longitude = metadata[flight].get(ATTR_LONGITUDE)
+                latitude = metadata[flight].get(ATTR_LATITUDE)
             else:
                 # Assume Flight has landed if missing.
                 altitude = 0
+                longitude = 0
+                latitude = 0
```

Comment:
(0,0) is a real place (off the coast of Ghana in the South Atlantic).  

https://www.google.co.uk/maps/place/0°00'00.0%22N+0°00'00.0%22E/

Is this what is wanted?  Would NaN be more appropriate here? (I don't know if that might break something else though)

Reply:
Hi @davet2001, 
thank you for your review and your feedback, that's a valid point, I've not considered. 
I've updated the Pull Request please have a look.

Reply:
‘None’ looks better (but like I said, I don’t know if that breaks something else!)